core/src/main/java/com/amplifyframework/datastore/DataStoreChannelEventName.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.datastore;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import com.amplifyframework.hub.HubCategory;
+import com.amplifyframework.hub.HubChannel;
+import com.amplifyframework.hub.HubEvent;
+
+import java.util.Objects;
+
+/**
+ * An enumeration of the names of events relating the the {@link DataStoreCategory},
+ * that are published via {@link HubCategory#publish(HubChannel, HubEvent)} on the
+ * {@link HubChannel#DATASTORE} channel.
+ */
+public enum DataStoreChannelEventName {
+    /**
+     * An item in the local storage has been published to the cloud.
+     * An event that uses this value for {@link HubEvent#getName()} will contain a model object
+     * in the {@link HubEvent#getData()}.
+     */
+    PUBLISHED_TO_CLOUD("published_to_cloud"),
```

Comment:
Another thing that we are looking at is "local only models" which do not get sync'd to the cloud. A few customers have been asking about this and @drochetti and Manuel are doing a design. In that case we might want to still publish to hub for state management in your app, so it's worth keeping that in mind for the list of possible events - e.g. `PUBLISHED_TO_LOCAL`

Reply:
Okay, cool. Yea, as you note, the `DataStoreChangeEventName` is really part of the API surface itself, since customers will engage with these names to understand the events on the Hub. I suspect this file will see a bunch more useful stuff come into it.